README.md

```diff
@@ -1 +1,89 @@
-# leaderboard-app
```

Comment:
- [ ] OPTIONAL
Your ReadMe file appears to have incorrect **markdown syntax** from the _Author section_ downwards. Kindly ensure to make your ReadMe more cleaner by using this following [this template](https://github.com/microverseinc/readme-template) but remember to customize it to your project.